src/witan/population/england/snpp_2018.clj

```diff
@@ -0,0 +1,47 @@
+(ns witan.population.england.snpp-2018
+  "This ns reads in the 2018 SNPP Population projection from 2018-2041
+  and converts it into a format suitable for use in the witan.send
+  model."
+  (:require [clojure.java.io :as io]
+            [tablecloth.api :as tc]
+            [tech.v3.dataset :as ds]
+            [witan.population.ncy :as ncy]))
+
+(def snpp-2018-file-name "2018snpppopulation/2018 SNPP Population persons.csv")
+
+(def output-columns [:calendar-year :academic-year :population])
+(def start-age 0)
+(def end-age 26)
```

Comment:
I wonder if you're going to "fix" these, should they be arguments to `snpp-2018->witan-send-population`? Not essential just a musing